module/main.go

```diff
@@ -1,2 +1,38 @@
 // Package main is a module with a cartographer SLAM service model.
 package main
+
+import (
+	"context"
+
+	"github.com/edaniels/golog"
+	viamcartographer "github.com/viamrobotics/viam-cartographer"
```

Comment:
For my knowledge, do you know why we need to alias this to `viamcartographer`, but didn't need add an alias for `golog`?

Reply:
I think the answer to your question is: 
`viam-cartographer` has a hyphen, go package names must be identifiers, identifiers cannot contain hyphens. golog does not have a hypen

Reply:
Ah! Thanks.